### PR TITLE
Add scrolling up and down div

### PR DESCRIPTION
**Pull request by ndufreche**
see #444

### CODE REVIEW

File: dist/css/bb-ui.css [hunk 1]
```diff
@@ -9929,22 +9929,27 @@ fieldset[disabled] #bb5-ui .btn-online.active {
...
   width: 100%;
   text-align: center;
-  opacity: 0.5;
+  opacity: 0.2;
   z-index: 999999;
 }
-#bb5-ui .scroll .fa,
-.scroll .fa {
+#bb5-ui .scroll .fa {
+  opacity: 0.6;
```
Comment (mickaelandrieu) [inline dist/css/bb-ui.css]: can you add the css rule for ie ?

``css
filter: alpha(opacity=60);
``

same for line 9937 :)

File: dist/css/bb-ui.css [hunk 2]
```diff
@@ -9929,22 +9929,27 @@ fieldset[disabled] #bb5-ui .btn-online.active {
...
   color: #f9f9f9;
-  font-size: 30px;
+  font-size: 50px;
+}
+#bb5-ui .scroll.over {
+  opacity: 0.6;
```
Comment (mickaelandrieu) [inline dist/css/bb-ui.css]: same here

File: src/tb/apps/content/components/DndManager.js [hunk 3]
```diff
@@ -108,6 +112,79 @@ define(
...
+
+                this.scrollFnc.size = false;
+
+                this.intervalId = setInterval(this.scrollFnc.bind(this.scrollFnc), 10);
+
+                document.querySelector('#bb5-ui').insertAdjacentHTML('beforeend', scrollUp + scrollDown);
+
+                scrolls = document.querySelectorAll('.scroll');
+
+                for (i = scrolls.length - 1; i >= 0; i = i - 1) {
```
Comment (mickaelandrieu) [inline src/tb/apps/content/components/DndManager.js]: I want a ``forEach`` <3

Comment (ndufreche) [inline src/tb/apps/content/components/DndManager.js]: Is not possible, forEach is for Array, in this case is a node collection.

File: dist/css/bb-ui.css (hunk 2 above)
Comment (ndufreche) [inline dist/css/bb-ui.css]: Integration upgrade, it's just a first and quick css implementation (all rules in fix.less have to be removed for 1.0, they are all temporally fixes)

File: dist/css/bb-ui.css (hunk 1 above)
Comment (ndufreche) [inline dist/css/bb-ui.css]: Integration upgrade, it's just a first and quick css implementation (all rules in fix.less have to be removed for 1.0, they are all temporally fixes)

File: src/tb/apps/content/components/DndManager.js (hunk 3 above)
Comment (mickaelandrieu) [inline src/tb/apps/content/components/DndManager.js]: :'( fair enough.

:+1:

Comment (fkroockmann) [inline src/tb/apps/content/components/DndManager.js]: You don't like jQuery :)

**Comment by fkroockmann** [comment]
:+1:

**Pull request merged**